app/src/components/Courses/ExploreCardPreview.tsx

```diff
@@ -167,7 +162,7 @@ const ExploreCardPreview: React.FC<ExploreCardPreviewProps> = ({ course }) => {
                 <div className="h-px bg-gray-200 w-full" />
 
                 {/* Description */}
-                {course.description && (
+                {course.description !== "" && (
```

Comment:
Do you make sure that description is never null or undefined?

Reply:
Course is initialized as a new course in the store "const [course, setCourse] = useState<Course>({title: "",
    description: "",
    category: "personal finance",
    difficulty: 1,
    status: "draft",
    estimatedHours: 0,
    sections: [],
    coverImg: "",
    _id: "0",
    creator: "",
    });" unless it's an existing one that is clicked onto and then the fetched data will set the course accordingly so description will never be undefined not according to typescripts type inference, or the implementation right now. Also a course cannot be saved anymore unless all required fields are filled out, so i dont expect it to be a problem. It was mostly just typescript that was raising noise which lead to that small change when we merged with dev.